src/code-templates/services/order-service/data-access-prisma/prisma/client/schema.prisma

```diff
@@ -0,0 +1,35 @@
+generator client {
+  provider = "prisma-client-js"
+  output   = "./client"
+}
+
+datasource db {
+  provider = "postgresql"
+  url      = "postgresql://myuser:myuserpassword@localhost:54320/shop?schema=public"
```

Comment:
Please add a comment for the user to change it

Reply:
Yes + I should put this under env var